js/apps/admin-ui/src/phaseII/orgs/useOrgFetcher.ts

```diff
@@ -504,6 +503,27 @@ export default function useOrgFetcher(realm: string) {
     }
   }
 
+  // POST /:realm/orgs/:orgId/idps/unlink
+  async function unlinkIDPtoOrg(orgId: OrgRepresentation["id"]) {
+    try {
+      const resp = await fetchPost(`${baseUrl}/orgs/${orgId}/idps/unlink`, {});
```

Comment:
Need to update to `/orgs/:orgId/idps/:idpAlias/unlink`

Reply:
Updated.